public/422.html

```diff
@@ -1,58 +1,150 @@
+  <div class="error-artwork-container">
+    <picture>
+      <source
+        srcset="/images/errors/small/404.png 576w,
+                /images/errors/medium/404.png 768w,
+                /images/errors/original/404.png 1200w">
+      <img src="/images/errors/small/404.png"/>
+    </picture>
+  </div>
+  <div class="text-center">
+    <h1>
+      <span class="text-primary font-weight-bold">422</span>
+      <small class="text-muted">Unprocessable Entity</small>
+    </h1>
+    <p>
+      Whatever your browser tried to send us, it was not right!
+    </p>
+    <p class="mb-0">
+      <i>Please retry what you initially tried, and if the problem persists, don't hesitate to contact us!</i>
+    </p>
+    <ul class="links mt-0">
+      <li>
+        <a href="https://twitter.com/retrospring">Twitter</a>
```

Comment:
might make sense to include the handle in the link text so people can immediately @ us from their preferred app?

```suggestion
        <a href="https://twitter.com/Retrospring">Twitter: @Retrospring</a>
```